### ISSUE TITLE
cannot import flake8 on pyodide

### ISSUE THREAD

**Issue by exKAZUu** [PyCQA/flake8#1562]
### how did you install flake8?

1. Open https://pyodide.org/en/stable/console.html
2. Run the following code
   ```   
   import micropip
   await micropip.install('flake8')
   from flake8.api import legacy as flake8
   ```
3. The following error is shown

```
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/lib/python3.9/site-packages/flake8/api/legacy.py", line 12, in <module>
    from flake8.main import application as app
  File "/lib/python3.9/site-packages/flake8/main/application.py", line 15, in <module>
    from flake8 import checker
  File "/lib/python3.9/site-packages/flake8/checker.py", line 661, in <module>
    ) -> Optional[multiprocessing.pool.Pool]:
AttributeError: 'NoneType' object has no attribute 'pool'
```

### unmodified output of `flake8 --bug-report`

We have no output because `pyodide` cannot import `flake8`.

#### what I expected to happen

`from flake8.api import legacy as flake8` should not throw any error.
To fix this issue, https://github.com/PyCQA/flake8/blob/main/src/flake8/checker.py#L595 should not refer to `multiprocessing.pool.Pool` because [pyodide](https://github.com/pyodide/pyodide) provides `multiprocessing.Pool` but does not provide `multiprocessing.pool.Pool`.

#### sample code

```python
import micropip
await micropip.install('flake8')
from flake8.api import legacy as flake8
```

#### commands ran

I've described whole commands at `how did you install flake8?`.

Note: I've also reported on https://github.com/pyodide/pyodide/issues/2210

**Comment by asottile** [comment]
dupe #1425
